service/src/main.rs

```diff
@@ -617,14 +616,14 @@ fn start_worker<E, T, D, InitializationHandler, WorkerModeProvider>(
 	println!("[+] [{:?}] Subscribed to events. waiting...", ParentchainId::Integritee);
 	loop {
 		if let Some(Ok(events)) = subscription.next_event::<RuntimeEvent, Hash>() {
-			print_events(events)
+			print_events(events, ParentchainId::Integritee)
 		}
 	}
 }
 
 fn init_target_parentchain<E>(
 	enclave: &Arc<E>,
-	tee_account_id: &AccountId32,
+	tee_account_id: Option<AccountId32>,
```

Comment:
This might be a bit too nitpicky, but as we only use `ref account` below, this can be an `Option<&AccountId>`